pom.xml

```diff
@@ -69,6 +69,24 @@
       <version>1.5</version>
     </dependency>
     <dependency>
+      <groupId>org.jenkins-ci.plugins</groupId>
+      <artifactId>subversion</artifactId>
+      <version>2.5</version>
```

Comment:
Is this to override some 1.x bundled dependency? I thought that would be obsolete as of core 2.x.

If this is because of a dependency from `workflow-scm-step`, then make sure _that_ is updated (newer versions do not depend on `subversion`), and remove this dependency.